### DISCUSSION TITLE
/usr/local/bin/containerd-rootless-setuptool.sh: line 58: realpath: command not found

### DISCUSSION THREAD

**Discussion by rickymunizjr** [containerd/nerdctl#3397] [Q&A] [answered]
Tried to install the full version and gave the following error:

$ sudo containerd-rootless-setuptool.sh install
/usr/local/bin/containerd-rootless-setuptool.sh: line 58: realpath: command not found

I am using macOS High Sierra 10.13.6 and cannot upgrade further, if there is any workaround with this version please let me know.

**Comment by AkihiroSuda** [comment, marked as answer]
`containerd-rootless-setuptool.sh` requires Linux.

On modern macOS you can use Lima to set up nerdctl, but I don't think it works on macOS 10.13. https://lima-vm.io/docs/examples/#running-containers
